### DISCUSSION TITLE
v2.5.3

### DISCUSSION THREAD

**Discussion by mashb1t** [lllyasviel/Fooocus#3437] [General]
## What's Changed
* fix: use weights_only for loading by @kit1980 in https://github.com/lllyasviel/Fooocus/pull/3427
* fix/feat: add checkbox and config to disable updating selected styles when describing an image by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/3430

## New Contributors
* @kit1980 made their first contribution in https://github.com/lllyasviel/Fooocus/pull/3427

**Full Changelog**: https://github.com/lllyasviel/Fooocus/compare/v2.5.2...v2.5.3

<hr /><em>This discussion was created from the release <a href='https://github.com/lllyasviel/Fooocus/releases/tag/v2.5.3'>v2.5.3</a>.</em>

**Comment by Rogala** [comment]
A warning appeared after the update, is this critical to performance?
`
e:\Fooocus\python_embeded\lib\site-packages\torch\_utils.py:831: UserWarning: TypedStorage is deprecated. It will be removed in the future and UntypedStorage will be the only storage class. This should only matter to you if you are using storages directly.  To access UntypedStorage directly, use tensor.untyped_storage() instead of tensor.storage()
  return self.fget.__get__(instance, owner)()
`

**Reply by kit1980** [reply]
That should be fine. The warning should go away in newer PyTorch version.

**Reply by mashb1t** [reply]
Yes, this warning isn't critical and has to do with the package updates in 2.5.0, all fine.

**Comment by TMFedd** [comment]
Appreciate all the great work you're doing!

How would one add default config for the new describe options?

Already added "describe_tab" as my default tab, what would be the config properties for Photograph or Anime/Art true/false and Apply Styles true/false?

**Reply by mashb1t** [reply]
Hey @TMFedd, you can find all possible config settings in the config example file

**Reply by TMFedd** [reply]
> Hey @TMFedd, you can find all possible config settings in the config example file

My mistake, wasn't aware that was being updated, should've checked first. Thanks though!

Got it set up, although the apply styles checkbox property name isn't very intuitive: "default_describe_apply_prompts_checkbox", shouldn't it be changed to "default_describe_apply_styles_checkbox" instead?

**Reply by mashb1t** [reply]
@TMFedd indeed, you're right. I'll put it on the list of the next release.
